website/content/docs/platform/k8s/vso/sources/hvs.mdx

```diff
@@ -73,10 +75,64 @@ spec:
     name: vso-app-secret
 ```
 
+### Static Secrets
+
+VSO supports syncing [static secrets](/hcp/docs/vault-secrets/static-secrets/create-static-secret)
+from an HCP Vault Secrets app to a Kubernetes Secret. VSO syncs the secrets to
+Kubernetes on the [refreshAfter][hvsa-spec] interval set in the
+HCPVaultSecretsApp spec.
+
+### Auto-rotating and Dynamic Secrets
+
+<Tip title="Feature availability">
+
+  VSO v0.9.0
+
+</Tip>
+
+VSO also supports syncing [auto-rotating](/hcp/docs/vault-secrets/auto-rotation)
+and [dynamic](/hcp/docs/vault-secrets/dynamic-secrets) secrets from an HCP Vault
+Secrets app to a Kubernetes Secret.
+
+VSO syncs auto-rotating secrets along with static secrets on the
+[refreshAfter][hvsa-spec] interval, and rotation is handled by HCP. VSO
+syncs dynamic secrets when the [specified percentage](/vault/docs/platform/k8s/vso/api-reference#hvsdynamicsyncconfig)
```

Comment:
nit could we add `/vault/docs/platform/k8s/vso/api-reference`  to https://github.com/hashicorp/vault/blob/137bba2b70c64261e5a04e214862d10116a055c1/website/content/partials/vso/common-links.mdx#L1

Not sure if there are any other "candidate" paths that we should consider for inclusion as common links.

Reply:
Do you have an example of how this would work? I'm not finding a way in markdown to define a link to the base url (`/vault/docs/platform/k8s/vso/api-reference`) and use that base link along with the anchor (`#hvsdynamicsyncconfig`) in the text.

Reply:
Maybe this one https://github.com/hashicorp/vault/blob/f5963cfe00d8ab4d62b2aabd699bd6e9ebf76034/website/content/docs/platform/k8s/vso/sources/vault/index.mdx?plain=1#L129

Reply:
Ah, then I misunderstood. Is the idea to use `common-links.mdx` for all VSO links, and not just links that are used on multiple pages? (The links in this PR are only used in a single page, which is why I didn't add them to common-links.)